app-containers/podman/files/disable-seccomp-4.6.2.patch

```diff
@@ -0,0 +1,10 @@
+--- a/Makefile
```

Comment:
Why?

Reply:
This is about TODO comments in src_compile()